fast64_internal/oot/oot_level_writer.py

```diff
@@ -301,8 +307,13 @@ def readRoomData(room, roomHeader, alternateRoomHeaders):
     room.disableSkybox = roomHeader.disableSkybox
     room.disableSunMoon = roomHeader.disableSunMoon
     room.echo = roomHeader.echo
-    room.objectList.extend([getCustomProperty(item, "objectID") for item in roomHeader.objectList])
-    if len(room.objectList) > 15:
+    room.objectIDList.extend(
+        [
+            room.getCustomIDFromKey(ootData.objectData.objectsByKey, obj, "objectKey", "objectID")
```

Comment:
I feel like this is a very complicated function call for a two-lines function (which can be one line), it would probably be simpler to read if you move the function's content here